packages/icons/src/svg/contains.svg

```diff
@@ -0,0 +1,8 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 16 16">
+  <g>
+    <defs>
+      <path id="contains-a" d="M4.391,7.536 C3.831,7.893 3.368,8.35 3,8.907 L4.15,9.614 C4.732,8.729 5.367,8.286 6.058,8.286 C6.625,8.286 7.192,8.571 7.758,9.143 C7.973,9.357 8.264,9.553 8.633,9.732 C9.001,9.911 9.444,10.001 9.965,10.001 C10.242,10.001 10.517,9.947 10.794,9.839 C11.069,9.732 11.337,9.589 11.598,9.411 C11.859,9.232 12.111,9.021 12.357,8.778 C12.602,8.536 12.816,8.286 13,8.029 L11.943,7.214 C11.559,7.686 11.222,8.036 10.931,8.264 C10.64,8.493 10.28,8.607 9.851,8.607 C9.437,8.607 8.915,8.335 8.287,7.793 C8.057,7.593 7.755,7.411 7.379,7.247 C7.005,7.083 6.548,7 6.011,7 C5.491,7 4.951,7.178 4.391,7.536 Z"/>
+    </defs>
+    <use fill="#020202" fill-rule="evenodd" xlink:href="#contains-a"/>
```

Comment:
please remove the need for id use and fill here.

Reply:
updated, thank you